release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1275,25 +1280,40 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|{sno-caps} cluster expansion with worker nodes
+| General Availability
+| General Availability
+| General Availability
+
+|{cgu-operator-first}
+|General Availability
+|General Availability
+|General Availability
+
+|NUMA-aware scheduling with NUMA Resources Operator
+|General Availability
+|General Availability
+|General Availability
+
 |Mount namespace encapsulation
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|General Availability
```

Comment:
According to the docs this feature is still TP 
https://github.com/openshift/openshift-docs/blob/enterprise-4.15/modules/enabling-encapsulation.adoc#configuring-mount-namespace-encapsulation

Reply:
Thank you @rsevilla87 !